api/resources/translations/messages-es.properties

```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
+login.unsupported_browser.outdated_cht_android = Hágales saber que actualice el cht-android.
+login.unsupported_browser.outdated_webview_apk = Hágales saber que actualice el Webview APK.
+login.unsupported_browser.outdated_browser = Hágales saber que actualice su navegador.
+message.characters.left = Quedan {{characters}} carácteres
+message.characters.left.multiple = {{messages}} SMS, quedan {{characters}} carácteres
+message.characters.left.multiple.many = {{messages}} SMS, quedan {{characters}} carácteres - \¡demasiados mensajes\!
+message.list.complete = No más mensajes
+messages.c.report_accepted = Gracias {{contact.name}} por registrar a {{patient_name}}. Su ID es {{patient_id}}. Han sido inscritos en el programa de salud infantil.
+messages.c.validation.months_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en meses y entre 0 y 59.
+messages.c.validation.patient_name = {{\#patient_name}}El formato de registro es incorrecto. Comience con C seguida de un espacio, edad en semanas seguida de un espacio y el nombre del niño (máximo de 30 caracteres).{{/patient_name}}{{^patient_name}}. Falta el nombre en el formato de registro. Comience con C seguida de un espacio, la edad en semanas seguida de un espacio y el nombre del niño {{/patient_name}}.
+messages.c.validation.weeks_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en semanas y entre 0 y 260.
+messages.d.report_accepted = Gracias {{contact.name}} por informar la entrega de {{patient_name}} ({{patient_id}}).
+messages.d.validation.days_since_birth = El informe de entrega de {{patient_id}} es incorrecto. Asegúrese de que los días transcurridos desde la entrega sean un número de días entre 0 y 365, o déjelo en blanco.
+messages.d.validation.delivery_code = {{\#delivery_code}}No se entendió el código de entrega. Utilice el código "F" para centro, "S" para parto en casa con partero calificado, "NS" para parto sin SBA.{{/delivery_code}}{{^delivery_code}}Faltaba el código de parto. Utilice el código "F" para centro, "S" para parto en casa con un asistente de parto calificado, "NS" para parto sin un asistente calificado.{{/delivery_code}}
 messages.errors.invalid = Mensaje inválido\:
 messages.errors.message.empty = Traducción no encontrada
 messages.errors.patient.missing = Paciente no encontrado
-messages.f.report_accepted =
-messages.f.report_accepted_parent = 
-messages.generic.no_provided_patient_id =
+messages.errors.place.missing = Lugar no encontrado
+messages.errors.unknown.contact = Este contacto ya no existe, no puedes enviarle un mensaje.
+messages.f.report_accepted = Gracias por marcar a {{patient_name}} ({{patient_id}}). Si se trata de una emergencia, envíela a la clínica de inmediato.
+messages.f.report_accepted_parent = {{patient_name}} ({{patient_id}}) fue marcado por {{contact.name}} ({{contact.phone}}) para mayor atención.
```

Comment:
What do you mean with "marcado"?
Could it be "puesto en contacto con"?

Reply:
haha yes, thanks!

Reply:
Ah, it says "flagged" like it added some sort of mark or label to the patient, so later, this person can get medical attention. 
So I guess we can use: `{{patient_name}} ({{patient_id}}) fue señalado por {{contact.name}} ({{contact.phone}}) para recibir mayor atención.` ?

Reply:
I made the change [in this commit ](https://github.com/medic/cht-core/pull/8980/commits/0a1c45c3ccf0ac06b676fba2702527aa6f2052d8) is it okay?